src/pandroid/app/src/main/java/com/panda3ds/pandroid/math/Vector2.java

```diff
@@ -0,0 +1,20 @@
+package com.panda3ds.pandroid.math;
+
+public class Vector2 {
+	public float x, y;
+	public Vector2(Vector2 value) { this(value.x, value.y); }
+
+	public Vector2(float x, float y) {
+		this.x = x;
+		this.y = y;
+	}
+
+	public float distanceTo(Vector2 vec) { return distance(x, y, vec.x, vec.y); }
```

Comment:
Can't this use function overloading instead of being called distanceTo?

Reply:
This can removed.
```suggestion
```